modules/UI/videolayout/RemoteVideo.js

```diff
@@ -164,7 +163,9 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let remoteControlState = null;
     let onRemoteControlToggle;
 
-    if (this._supportsRemoteControl) {
+    if (this._supportsRemoteControl
+        && ((!APP.remoteControl.active && !this._isRemoteControlSessionActive)
+            || APP.remoteControl.controller.activeParticipant === this.id)) {
```

Comment:
Is it not possible to get this state from redux? Is any of this state available in redux?